offchain/hello-world-api/src/Signing.purs

```diff
@@ -0,0 +1,58 @@
+module Signing
+  ( getServerPubKey
+  , serverSignTx
+  , getServerCmd
```

Comment:
Because we're not using a signing server anymore, can we remove all mentions of it in the code?